service/src/test/java/bio/terra/catalog/service/DatasetServiceTest.java

```diff
@@ -177,71 +178,69 @@ void listDatasetsUsingAdminPermissions() {
   @Test()
   void testDeleteMetadataWithInvalidUser() {
     mockDataset();
-    assertThrows(UnauthorizedException.class, () -> datasetService.deleteMetadata(user, datasetId));
+    assertThrows(UnauthorizedException.class, () -> datasetService.deleteMetadata(datasetId));
   }
 
   @Test()
   void testDeleteMetadata() {
     mockDataset();
-    when(samService.hasGlobalAction(user, SamAction.DELETE_ANY_METADATA)).thenReturn(true);
-    datasetService.deleteMetadata(user, datasetId);
+    when(samService.hasGlobalAction(SamAction.DELETE_ANY_METADATA)).thenReturn(true);
+    datasetService.deleteMetadata(datasetId);
     verify(datasetDao).delete(dataset);
   }
 
   @Test
   void testGetMetadataWithInvalidUser() {
     mockDataset();
-    assertThrows(UnauthorizedException.class, () -> datasetService.getMetadata(user, datasetId));
+    assertThrows(UnauthorizedException.class, () -> datasetService.getMetadata(datasetId));
   }
 
   @Test
   void testGetMetadata() {
     mockDataset();
-    when(samService.hasGlobalAction(user, SamAction.READ_ANY_METADATA)).thenReturn(true);
-    datasetService.getMetadata(user, datasetId);
-    verify(datasetDao).retrieve(datasetId);
+    when(samService.hasGlobalAction(SamAction.READ_ANY_METADATA)).thenReturn(true);
+    assertThat(
+        datasetService.getMetadata(datasetId), is(METADATA_WITH_ID.formatted(datasetId.uuid())));
   }
 
   @Test
   void testGetMetadataUsingTdrPermission() {
-    reset(datasetDao);
-    when(datasetDao.retrieve(datasetId)).thenReturn(tdrDataset);
-    when(datarepoService.getRole(user, sourceId)).thenReturn(DatasetAccessLevel.READER);
-    datasetService.getMetadata(user, datasetId);
-    verify(datasetDao).retrieve(datasetId);
+    when(datasetDao.retrieve(tdrDataset.id())).thenReturn(tdrDataset);
+    when(datarepoService.getRole(sourceId)).thenReturn(DatasetAccessLevel.READER);
+    assertThat(
+        datasetService.getMetadata(tdrDataset.id()),
+        is(METADATA_WITH_ID.formatted(tdrDataset.id().uuid())));
```

Comment:
Optional Nit: I personally prefer working with object structures rather than formatted strings, because then order doesn't matter for things like `.formatted`. Would it be possible to have `METADATA_WITH_ID` be an object, and hold off stringifying it until we get to this point here?